### DISCUSSION TITLE
Ignore BBCode for RichTextLabel auto translate

### DISCUSSION THREAD

**Discussion by Joy-less** [godotengine/godot-proposals#9600] [GUI]
As far as I know, the only easy way to center a `RichTextLabel` is to use the `[center]` BBCode tag:
```
[center]Text
```
However, this means `Text` won't be auto-translated unless the translation table includes the `[center]` tag.
Tags aren't rendered, so they should be excluded from auto translation.

**Comment by Calinou** [comment]
In this case, I recommend concatenating the text with a script as follows:

```gdscript
func _ready() -> void:
	text = "[center]" + tr("Text")
```

Either way, I feel having a default alignment property in RichTextLabel would make more sense.
